src/external/time.rs

```diff
@@ -29,4 +30,91 @@ impl TimeApi {
                 Ok(result) => Ok(result),
             }
     }
+
+    #[allow(dead_code)]
+    pub fn date_time_string_to_u64_timestamp(
+        date_time_str: &str,
+        format_str: Option<&str>,
+    ) -> Option<i64> {
```

Comment:
This returns i64, but the function says u64.
I think if you return `Some(datetime_utc.timestamp() as u64)` it should work